backend/sim/wallet/address.go

```diff
@@ -111,22 +107,12 @@ func (a *Address) MarshalBinary() ([]byte, error) {
 	return data[:], nil
 }
 
-// Decode decodes an address from an io.Reader. Part of the
-// go-perun/pkg/io.Serializer interface.
-func (a *Address) Decode(r io.Reader) error {
-	var length int64
-	err := perunio.Decode(r, &length)
-	if err != nil {
-		return err
-	}
-	if length != 64 {
-		return fmt.Errorf("unexpected address length %d, want %d", length, common.AddressLength)
+// UnmarshalBinary unmarshalled the address from its binary representation.
+func (a *Address) UnmarshalBinary(data []byte) error {
+	if len(data) != 64 {
```

Comment:
Use of magic number. Please create a const for that to give some context. Also see linter https://github.com/tommy-muehle/go-mnd.

Reply:
Update: I realize that the linter is not enabled here yet. This was done concurrently.

Reply:
Updated in c3716d51027f23468796c574881805e90d3c6ac6.